File: apps/s-2-admin/src/api/workflowReminder/WorkflowReminderWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";
import { DateTimeFilter } from "../../util/DateTimeFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { BooleanFilter } from "../../util/BooleanFilter";
import { WorkflowStepWhereUniqueInput } from "../workflowStep/WorkflowStepWhereUniqueInput";

export type WorkflowReminderWhereInput = {
  id?: IntFilter;
  booking?: BookingWhereUniqueInput;
  method?: "EMAIL" | "SMS";
  scheduledDate?: DateTimeFilter;
  referenceId?: StringNullableFilter;
  scheduled?: BooleanFilter;
  workflowStep?: WorkflowStepWhereUniqueInput;
};

File: apps/cactus-demo-dotnet/src/APIs/WorkflowReminder/Dtos/WorkflowReminderWhereInput.cs
using CactusDemoDotnet.Core.Enums;

namespace CactusDemoDotnet.APIs.Dtos;

public class WorkflowReminderWhereInput
{
    public int? Id { get; set; }

    public int? Booking { get; set; }

    public MethodEnum? Method { get; set; }

    public DateTime? ScheduledDate { get; set; }

    public string? ReferenceId { get; set; }

    public bool? Scheduled { get; set; }

    public int? WorkflowStep { get; set; }
}
